### PR TITLE
Filter out assets that are not available for swap from asset selector

### LINKED ISSUE #485: Filter out LpNft's from swap asset selector
When selecting an asset to swap, we see LpNfts included:

Per @grod220 's [comment](https://github.com/penumbra-zone/web/pull/476#pullrequestreview-1870071861):

> We should not have lpnft's in that selector drop down box at all.

This ticket is to filter for only spendable assets wherever we have asset selectors. We'll likely want to use [the `assetPatterns` const](https://github.com/penumbra-zone/web/blob/5f5e98bf6e85dc58f19df83977f6aee748a6ca87/packages/constants/assets/index.ts).

### PR DESCRIPTION
**Pull request by Valentine1898**
close #485

### CODE REVIEW

**Comment by Valentine1898** [comment]
image
image

I think we should filter not only `lpNft` but also `proposalNft`, `unbondingToken`, `votingReceipt`, obviously they can't be swapped.

File: apps/webapp/src/components/swap/swap-loader.tsx [hunk 1]
```diff
@@ -26,13 +31,18 @@ export interface SwapLoaderResponse {
 const getAndSetDefaultAssetBalances = async () => {
   const assetBalances = await getBalances();
 
+  // filter assets that are not available for swap
+  const filteredAssetBalances = assetBalances.filter(b =>
+      [assetPatterns.lpNft, assetPatterns.proposalNft, assetPatterns.unbondingToken, assetPatterns.votingReceipt]
```
Comment (grod220) [inline apps/webapp/src/components/swap/swap-loader.tsx:36]: I think there could potentially be a market for unbonding tokens. Think we should keep that one.

Comment (Valentine1898) [reply apps/webapp/src/components/swap/swap-loader.tsx]: It's actually kind of weird. 
We will have 100 different liquid staking tokens for 100 different validators, and for each of these liquid staking tokens we can have many more unbonding tokens for each epoch  :exploding_head:

Comment (Valentine1898) [reply apps/webapp/src/components/swap/swap-loader.tsx]: But in general I agree, removed unbonding tokens from filtering

**Review by grod220 — APPROVED** [review]

**Pull request merged**